scripts/check_containers.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Check if there were any dockerfiles modified in the latest pull request.
+
+diff_latest() {
+  git diff --name-only "$(git log -1 --merges --pretty=format:%H)"..HEAD
+}
+
+# Copy the diff output to stderr to list the modified files
+if diff_latest | tee /dev/stderr | grep -i dockerfile > /dev/null
```

Comment:
I think the `if ... | grep >/dev/null` can be written `if ... | grep -q`

Reply:
Your choice whether to make that change or not.

Reply:
Looks better with `-q` I think, added that :+1: